src/Pages/juheejang/Login/Login.js

```diff
@@ -32,7 +32,7 @@ class Login extends React.Component {
       pwValue: value,
     })
 
-    const isValPw = value.includes("@") && event.target.value.length >= 5;
+    const isValPw = this.state.idValue.includes("@") && value.length >= 5;
     this.setState({
        loginBtnOpacity: isValPw ? true : false,
        loginBtn: isValPw ? false : true
```

Comment:
주희님 안녕하세요! :D 3팀에서 놀러왔습니다! ㅎㅎ 이미 너무 깔끔하게 잘하셨지만 오전에 관희님이 말씀하신 것처럼 아래와 같이 더 줄일 수도 있을 것 같아요!
```suggestion
    this.setState({
       loginBtnOpacity: isValPw,
       loginBtn: !isValPw
```

Reply:
다시 보니 이미 위에 관희님이 리뷰 남기셨군요ㅋㅋㅋㅋㅋ

Reply:
handleIdInput과 handlePwInput도 name을 활용해서 하나로 만들어줄 수 있을 것 같아요! 

```
handleInput = (event) => {
    const { name, value } = event.target;
    this.setState({ [name]: value };
  };
```
대신 각각의 name 속성을 state와 맞춰서 `idValue`, `pwValue`로 수정만 해주면 되겠네요!

코드카타 짝꿍 코드 잘 보고 갑니다! ^0^

Reply:
지형님! 코드리뷰 감사합니다 😀
id 값과 pw 값을 받는 함수를 따로 작성해서 각각 조건문을 실행하는 것 보다, 말씀하신대로 name을 이용해서 handleInput 함수 하나로 state의 값을 변경시킬 수 있도록 하니 코드가 더욱 짧고 간결해졌네요 :) 
수정했습니다 👍